xbmc/XBDateTime.cpp

```diff
@@ -1482,6 +1482,8 @@ std::string CDateTime::GetAsLocalizedTime(TIME_FORMAT format, bool withSeconds /
       return GetAsLocalizedTime("mm:ss", true);
     case TIME_FORMAT_HH:  // this forces it to a 12 hour clock
       return GetAsLocalizedTime(use12hourclock ? "h" : "HH", false);
+    case TIME_FORMAT_HH_SS:
```

Comment:
Are these used by skinners (maybe through `CGUIInfoManager`)? If so does it require changes there to make it available?

Reply:
I don't believe so. The time format [already exists](https://github.com/xbmc/xbmc/blob/master/xbmc/XBDateTime.h#L58), it just didn't exist here for the `GetAsLocalizedTime` method